server/lib/mod-utils.js

```diff
@@ -163,7 +164,7 @@
       item.push(
         {
           html: '<a href="/pool-management/pool-overview/' +
-              pool.poolNumber + '" class="govuk-link">' + pool.poolNumber + '</a>',
+              pool.poolNumber + '?status=' + queryStatus + '" class="govuk-link">' + pool.poolNumber + '</a>',
```

Comment:
Should probably handle within the pool overview controller instead.
If no status provided default to 'responded' or court variant.

That way no matter where we navigate to the pool overview from it will correctly show responded people first (there are too many links to pool overview in the app to change).

Along with this, the radio buttons do not check correctly if no status provided, they should default to responded.

The same goes for pagination, if there is a status filter provided this needs to be carried through pagination, this does not happen with current code. Should be able to pass the status filter into the page items to handle this.

Reply:
It's problematic as no status is interprated as all when the default should be something else. So I can't easily tell if it's the first visit to the page or the user has selected all on purpose. I can work around it but it will affect the court page as well and was starting to snowball a little. The pool overview page itself has ?status=created when you navigate to it so I thought this would be a decent approach. I can take another look but it will take time.